references/classification/train.py

```diff
@@ -68,6 +82,14 @@ def _get_cache_path(filepath):
 
 
 def main(args):
+    if args.apex:
+        if (sys.version_info < (3, 0)):
+            print("Apex currently only supports Python 3. Aborting.")
```

Comment:
nit: you can raise an error in this case
```python
raise RuntimeError("Apex currently only supports Python 3. Aborting.")
```

Reply:
yep very good suggestion. I've edited this.